### PR TITLE
[bug 1253686] Handle no total_pages from SurveyGizmo API

### PR DESCRIPTION
**Pull request by rehandalal**
small r?

### CODE REVIEW

File: kitsune/kpi/cron.py [hunk 1]
```diff
@@ -632,7 +632,7 @@ def calculate_csat_metrics():
             timeout=300)
 
         results = json.loads(response.content)
-        total_pages = results['total_pages']
+        total_pages = results['total_pages'] if 'total_pages' in results else 1
```
Comment (safwanrahman) [inline kitsune/kpi/cron.py]: I think its better written as
``results.get('total_pages', 1)``
What do you think?

**Comment by mythmon** [comment]
I think @safwanrahman is right. `.get(key, default_val)` is more idiomatic and easier to read. Besides that, this should fix the problem, so r+wc.

**Comment by rehandalal** [comment]
fixed ^

**Comment by safwanrahman** [comment]
r+

**Pull request merged**